### PR TITLE
Remove VPA for the csi-driver-node and csi-driver-manila-node DaemonSets

### PR DESCRIPTION
**Pull request by ialidzhikov**
**How to categorize this PR?**

/area auto-scaling
/kind enhancement
/platform openstack

**What this PR does / why we need it**:
Similar to https://github.com/gardener/gardener-extension-provider-gcp/pull/929

**Which issue(s) this PR fixes**:
N/A

**Special notes for your reviewer**:
N/A

**Release note**:

```noteworthy user
The kube-system/csi-driver-node and kube-system/csi-driver-manila-node DaemonSets are no longer scaled by a VerticalPodAutoscaler as they do not really benefit from it. Removing VerticalPodAutoscaler for these components will reduce unnecessary evictions by VPA and will be a mitigation for https://issues.k8s.io/126921.
```

### CODE REVIEW

**Comment by ialidzhikov** [comment]
I am sorry for all the back and forth with PR (marking ready for review and then draft).

tl;dr: It should be safe to remove VPA for the csi node plugin DaemonSet. The PR is ready for review!

**Review by hebelsan — APPROVED** [review]
/lgtm
